### PR TITLE
Quests models

### PR DESCRIPTION
**Pull request by pakokrew**
Implementing #290 

Creation of the base model for quests.

### CODE REVIEW

File: hasura/metadata/tables.yaml [hunk 1]
```diff
@@ -343,6 +400,169 @@
...
+      - cooldown
+      - description
+      - external_link
+      - repetition
+      - status
+      - title
+      filter:
+        created_by_player_id:
+          _eq: X-Hasura-User-Id
+      check: null
```
Comment (alalonde) [inline hasura/metadata/tables.yaml:476]: what is `check`?

**Review by alalonde — APPROVED** [review]
looks good as an initial pass

File: hasura/metadata/tables.yaml (hunk 1 above)
Comment (pakokrew) [reply hasura/metadata/tables.yaml:476]: I don't know, this is auto generated by hasura. Maybe it is the "post-update" check.

Comment (hammadj) [reply hasura/metadata/tables.yaml:476]: Yep thats what it is

**Review by hammadj — APPROVED** [review]

**Pull request merged**